ext/ftp/php_ftp.c

```diff
@@ -318,12 +318,14 @@ static void ftp_destructor_ftpbuf(zend_resource *rsrc)
 PHP_MINIT_FUNCTION(ftp)
 {
 #ifdef HAVE_FTP_SSL
+#if PHP_OPENSSL_API_VERSION < 0x10100
```

Comment:
I don't think this constant is available here, as it's part of php_openssl.h, which is not included here (and I think also shouldn't be included for dependency reasons). This check should be done using vanilla openssl constants instead.

Reply:
The SSL code enabling code seems differs a lot from Unix to Windows; On Unix this seems to depend on `ext/openssl` being enabled, whereas on Windows it just depends the OpenSSL library. I suppose this was made in the effort to actually make `ftp_ssl_connect()` available on Windows.

Either way @nikic is correct; this needs another way of doing it, at least to make it work on Windows builds like `configure --disable-all --enable-cli --enable-ftp=shared`

Reply:
Fixed.